test/components/entity/show.spec.js

```diff
@@ -90,7 +90,7 @@ describe('EntityShow', () => {
           return testData.standardEntities.last();
         })
         .respondWithData(() => testData.extendedAudits.sorted())
-        .respondWithData(() => []);
+        .respondWithData(() => testData.extendedEntityVersions.sorted());
```

Comment:
Same as above: this was diffs before and should have been changed as part of #880.